app/Http/Controllers/Setup/Controller.php

```diff
@@ -125,17 +141,33 @@ public function save_auth_settings() {
     public function save_config() {
         $config = file_get_contents(__DIR__.'/../../../../.env.example');
 
-        // Generate an APP_KEY
-        $app_key = 'base64:'.base64_encode(Encrypter::generateKey('AES-256-CBC'));
-        self::write_config_value($config, 'APP_KEY', $app_key);
-
-        self::write_config_value($config, 'APP_DEBUG', 'false');
-        self::write_config_value($config, 'APP_ENV', 'production');
+        // Pull from existing ENV
+        foreach([
```

Comment:
I explicitly listed the options here that would be configured by the installer, and didn't add all the app-level options to the installer so that I could reconfigure this later. I would rather have people configure these in the installer rather than in the Heroku deploy stage.

Reply:
This specific section, even without app.json I really like as it cascades config a power-user might set to enable skipping further manual installer steps.

Outside of testing this PR, I'd never want to use an installer or play with the `.env` text files